### PR TITLE
Add Podcasts by @Karambirov

### PR DESCRIPTION
**Pull request by Karambirov**
1. [x] Project URL: https://github.com/Karambirov/Podcasts
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17554,6 +17554,34 @@
...
+        "media", 
+        "rxswift"
+      ],
+      "description": "A clone of Apple's Podcasts",
+      "source": "https://github.com/Karambirov/Podcasts",
+      "license": "mit",
+      "tags": [
+        "swift", 
+        "rxswift", 
+        "itunes"
```
Comment (dkhamsing) [inline contents.json]: Can you remove itunes - that’s not a tag we usually track

File: contents.json [hunk 2]
```diff
@@ -17554,6 +17554,34 @@
       "date_added": "Mar 29 2019",
       "suggested_by": "@dkhamsing",
       "stars": 287
+    },
+    {
+      "title": "Podcasts",
+      "category-ids": [
+        "audio", 
+        "clone", 
+        "media",
```
Comment (dkhamsing) [inline contents.json]: Can you remove media, audio is under media already

Comment (Karambirov) [reply contents.json]: Done

File: contents.json (hunk 1 above)
Comment (Karambirov) [reply contents.json]: Done

**Comment by dkhamsing** [comment]
Thank you 🙏

**Pull request merged**